src/dmd/traits.d

```diff
@@ -1989,3 +1880,128 @@ Lnext:
         e.error("unrecognized trait `%s`", e.ident.toChars());
     return new ErrorExp();
 }
+
+// compare arguments of __traits(isSame)
```

Comment:
```suggestion
/// compare arguments of __traits(isSame)
```

Reply:
it's private so more a code comment than a documentation one